pkg/reconciler/autoscaling/kpa/kpa.go

```diff
@@ -277,3 +277,16 @@ func activeThreshold(pa *pav1alpha1.PodAutoscaler) int {
 
 	return int(min)
 }
+
+// resolveScrapeTarget returns metric service name to be scraped based on TBC configuration
+// TBC == -1 => activator in path, don't scrape the service
+func resolveScrapeTarget(ctx context.Context, pa *pav1alpha1.PodAutoscaler, metricSN string) string {
+	tbc := config.FromContext(ctx).Autoscaler.TargetBurstCapacity
+	if v, ok := pa.TargetBC(); ok {
+		tbc = v
+	}
```

Comment:
To minimize work we do, how about...
```suggestion
	tbc := 0.
	if v, ok := pa.TargetBC(); ok {
		tbc = v
	} else {
	         tbc = config.FromContext(ctx).Autoscaler.TargetBurstCapacity
	}
```

Reply:
Fixed in 7b1feced1c20f12f662502628344d8087c3978f6.